sample.wdl

```diff
@@ -125,6 +126,19 @@ workflow Sample {
         metrics.picardMetricsFiles, metrics.rnaMetrics,
         metrics.targetedPcrMetrics, [markdup.metricsFile], flatten(qc.reports)]
 
+    call structuralVariantCalling.SVcalling as svCalling{
+        input:
+            bamFile = bqsr.recalibratedBam,
+            bamIndex = bqsr.recalibratedBamIndex,
+            referenceFasta = referenceFasta,
+            referenceFastaFai = referenceFastaFai,
+            referenceFastaDict = referenceFastaDict,
+            bwaIndex = bwaIndex,
+            sample = sample.id,
+            outputDir = sampleDir,
+            dockerImages = dockerImages
+    }
+
     output {
```

Comment:
I'm guessing there is some output which should get passed along to the top-level pipeline outputs, otherwise, they won't be included in the output directory produced by cromwell.